compartmentalize-grains-via-pop.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: compartmentalize-grains-via pop
+- Start Date: 2020-06-22
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Grains is currently a static component of salt.  We want it to be developed in a more compartmentalized fashion.
+The engineers at saltstack have created kernel-specific projects that functionally replace the grains
```

Comment:
Does this mean OS kernel, or is it another food-based pun?

Reply:
It means OS kernel :+1: